RecoLocalCalo/HcalRecProducers/python/HBHEMethod2Parameters_cfi.py

```diff
@@ -11,7 +11,7 @@
     ts4Max                = cms.vdouble(100.,70000.), #fC # this is roughly 20 GeV
     pulseJitter           = cms.double(1.),           #GeV/bin 
     ###
-    meanTime              = cms.double(0.),   #ns 
+    meanTime              = cms.double(2.),   #ns 
```

Comment:
@slava77
I added one slide on the M2 timing jetHT data 2016 
https://indico.cern.ch/event/570231/contributions/2306435/attachments/1341696/2021200/PR15959.pdf 
Data and MC just behave differently.
So the data are aligned such as the M2 timing is at zero, when moving the initial parameter of the fit,  the fitted time goes up.
The MC instead prefer the 2ns. 
So probably the best things for now would be not to move this parameter and tackle this time shift difference in MC at a more fundamental level (if it can be done).
Let me know if I should remove this commit c847539 from this PR.